packages/flutter_tools/test/commands.shard/hermetic/build_web_test.dart

```diff
@@ -299,29 +299,7 @@ void main() {
       FileSystem: () => fileSystem,
       FeatureFlags: () => TestFeatureFlags(isWebEnabled: true),
       ProcessManager: () => processManager,
-      BuildSystem:
-          () => TestBuildSystem.all(BuildResult(success: true), (
-            Target target,
-            Environment environment,
-          ) {
-            expect(environment.defines, <String, String>{
```

Comment:
`run` is not called above, so this section is NEVER called. I put in a throw. It's a no-op.

Reply:
Ah that makes sense 😊